diff -ru lpme.orig/src/lpme_common.cpp lpme/src/lpme_common.cpp
--- lpme.orig/src/lpme_common.cpp	2014-11-04 19:14:57.000000000 -0600
+++ lpme/src/lpme_common.cpp	2014-12-13 15:46:10.239290706 -0600
@@ -161,7 +161,7 @@
   // Make arma objects
   arma::vec gWin(gWinput.begin(), m, false);
   arma::vec fWin(fWinput.begin(), m, false);
-  arma::vec mcon(mconst.begin(), m, false);
+  arma::vec mcon(const_cast<NumericVector&>(mconst).begin(), m, false);
   
   // FFT for fW
   arma::vec Xin=mcon%fWin;
@@ -228,7 +228,7 @@
   // Make arma objects
   arma::vec gWin(gWinput.begin(), m, false);
   arma::vec fWin(fWinput.begin(), m, false);
-  arma::vec mcon(mconst.begin(), m, false);
+  arma::vec mcon(const_cast<NumericVector&>(mconst).begin(), m, false);
   
   // FFT for fW
   arma::vec Xin=mcon%fWin;
