diff -ru lme4.orig/DESCRIPTION lme4/DESCRIPTION
--- lme4.orig/DESCRIPTION	2013-10-25 14:06:19.000000000 -0500
+++ lme4/DESCRIPTION	2014-01-22 20:59:09.390416913 -0600
@@ -16,10 +16,9 @@
     Eigen C++ library for numerical linear algebra and RcppEigen "glue".
 Depends: R (>= 2.14.0), lattice, Matrix (>= 1.1), methods, stats
 LinkingTo: Rcpp, RcppEigen
-Imports: graphics, grid, splines, MASS, nlme, minqa (>= 1.1.15)
+Imports: graphics, grid, splines, MASS, nlme, minqa (>= 1.1.15), Rcpp
 Suggests: boot, PKPDmodels, MEMSS, testthat, ggplot2, mlmRev, optimx
-        (>= 2013.8.6), plyr, reshape, Rcpp (>= 0.10.1), RcppEigen (>=
-        0.3.1.2.3)
+        (>= 2013.8.6), plyr, reshape, RcppEigen (>= 0.3.1.2.3)
 LazyData: yes
 License: GPL (>= 2)
 URL: https://github.com/lme4/lme4/ http://lme4.r-forge.r-project.org/
diff -ru lme4.orig/NAMESPACE lme4/NAMESPACE
--- lme4.orig/NAMESPACE	2013-10-25 08:17:01.000000000 -0500
+++ lme4/NAMESPACE	2014-01-22 20:58:50.030417479 -0600
@@ -20,6 +20,7 @@
 importMethodsFrom("Matrix", "%*%",coerce,crossprod,diag,t,tcrossprod)
 ## other CRAN packages:
 importFrom("minqa", bobyqa)
+importFrom("Rcpp", evalCpp)
 
 export(bootMer)
 export(devcomp)
