diff -ru mirtCAT.orig/src/CAT_functions.cpp mirtCAT/src/CAT_functions.cpp
--- mirtCAT.orig/src/CAT_functions.cpp	2015-01-22 13:26:17.000000000 -0600
+++ mirtCAT/src/CAT_functions.cpp	2015-01-23 06:54:46.346579157 -0600
@@ -38,7 +38,7 @@
     const List pars(Rpars);
     const vector<double> Theta = as< vector<double> >(RTheta);
     const vector<int> which_not_answered = as< vector<int> >(Rwhich_not_answered);
-    const NumericVector rw(Rw);
+    /*const*/ NumericVector rw(Rw);
     const int criteria = as<int>(Rcriteria);
     const int len = which_not_answered.size();
     const int nfact = Theta.size();
