diff -ru DNAprofiles.orig/src/dbcomparepairwise.cpp DNAprofiles/src/dbcomparepairwise.cpp
--- DNAprofiles.orig/src/dbcomparepairwise.cpp	2015-04-21 02:15:28.000000000 -0500
+++ DNAprofiles/src/dbcomparepairwise.cpp	2016-12-29 06:36:08.057248658 -0600
@@ -1,4 +1,6 @@
 // [[Rcpp::depends(RcppProgress)]]
+#include <Rcpp.h>
+using namespace Rcpp;
 #include <progress.hpp>
 #include <vector>
 
Only in DNAprofiles/src: symbols.rds
